plugins/qtosgrave/osglodlabel.cpp

```diff
@@ -42,7 +42,7 @@ OSGLODLabel::OSGLODLabel(const std::string& label, const RaveVector<float>& colo
 
     // Set up other text element properties
     text->setText(label);
-    text->setCharacterSize(0.05);
+    text->setCharacterSize(0.05 * scale);
```

Comment:
How does scaling the arguments in the `text->setFontResolution` below here by `scale` as opposed to keeping them constant look?

Reply:
i think we should expose the size as it is in the osg to the API. that's more intuitive for user to call. the height is in the same scale of the object rendered in the viewer. we can make the default height 0.05 then it is 50 mm height in the environment.


```suggestion
    text->setCharacterSize(height);
```